build-system/test-configs/forbidden-terms.js

```diff
@@ -810,6 +810,7 @@ const forbiddenTermsSrcInclusive = {
   '\\.pageXOffset(?!_)': bannedTermsHelpString,
   '\\.pageYOffset(?!_)': bannedTermsHelpString,
   '\\.innerWidth(?!_)': bannedTermsHelpString,
+  '\\.toggleAttribute(?!_)': 'please use `toggleAttribute()` from core/dom',
```

Comment:
@jridgewell I feel like we should add this to forbidden terms to prevent future errors. WDYT?

Reply:
SGTM.